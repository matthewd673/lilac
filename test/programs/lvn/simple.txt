i32 a = b * c
i32 d = b
i32 e = d * c
