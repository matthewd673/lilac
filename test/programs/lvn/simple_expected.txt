i32 a = b * c
i32 d = b
i32 e = a
